tests/Unit/Domain/Condition/Period/PeriodTest.php

```diff
@@ -16,17 +16,36 @@ public function testGetters(): void
         $measure = $this->createMock(Measure::class);
         $start = new \DateTimeImmutable('2022-12-20 09:00:00');
         $end = new \DateTimeImmutable('2022-12-20 18:00:00');
+        $days = [
+            ApplicableDayEnum::MONDAY->value,
+            ApplicableDayEnum::TUESDAY->value,
+            ApplicableDayEnum::WEDNESDAY->value,
+            ApplicableDayEnum::FRIDAY->value,
+        ];
+
         $period = new Period(
             '9f3cbc01-8dbe-4306-9912-91c8d88e194f',
             $measure,
             true,
-            [ApplicableDayEnum::MONDAY, ApplicableDayEnum::THURSDAY],
+            $days,
             $start,
             $end,
         );
 
+        $daysRange = [
```

Comment:
Je pense qu'il faudrait "_unit test the hell of out `getDaysRanges()`_"

Or en l'insérant dans un test qui fait un peu tout, j'ai peur qu'on oublie des choses. Ou plutôt : il n'est pas facile de voir si on oublie un cas ou pas.

On devrait donc à mon avis avoir un ou des tests dédiés qui vérifient tous les cas de façon méthodique

* 1 jour coché
* 2 jours consécutifs cochés
* 3 jours consécutifs cochés
* 1 jour coché, puis 2 jours consécutifs cochés
* 1 jour coché, puis 1 jour séparé coché, puis 2 jours consécutifs cochés
* Lundi et dimanche cochés
* Tous les jours cochés